dev/ToastNotifications/ToastProgressDataABI.cpp

```diff
@@ -0,0 +1,69 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+#include "pch.h"
+
+#include "ToastProgressData.h"
+#include "ToastProgressDataABI.h"
+#include "ToastNotificationUtility.h"
+
+namespace winrt
+{
+    using namespace winrt::Microsoft::Windows::ToastNotifications;
+}
+
+namespace ToastABI
+{
+    using namespace ::ABI::Microsoft::Internal::ToastNotifications;
+}
+
+ToastProgressDataABI::ToastProgressDataABI(winrt::ToastProgressData const& progressData)
+{
+    m_progressData = progressData;
+}
+
+STDMETHODIMP_(HRESULT __stdcall) ToastProgressDataABI::get_SequenceNumber(_Out_ unsigned int* value)
```

Comment:
Should be just `STDMETHODIMP`

`STDMETHODIMP_(HRESULT __stdcall)` has 2 problems:

1. `STDMETHOD_(HRESULT)` is redundant with `STDMETHODIMP`. STDMETHOD_(type) is for when you don't return an HRESULT e.g. `STDMETHOD_(void)`
2. STDMETHODIMP_() already includes __stdcall

See winnrt.h
```
#define STDAPICALLTYPE          __stdcall
...
#define STDMETHODIMP            HRESULT STDMETHODCALLTYPE
#define STDMETHODIMP_(type)     type STDMETHODCALLTYPE
```

Thus...

* `STDMETHODIMP` == `HRESULT __stdcall`
* `STDMETHODIMP_(x)` == `x __stdcall`